pycharm_debugger.py

```diff
@@ -0,0 +1,12 @@
+def set_trace():
+    import pydevd_pycharm
+
+    gateway_ip = "172.17.0.1"
+    # Set up the debugger to connect to the host via the gateway IP
+    pydevd_pycharm.settrace(
+        host=gateway_ip,
+        port=12345,  # Use the same port number configured in PyCharm
```

Comment:
Should this be an environment variable?

Reply:
This shouldn't change as far as I'm aware. it's equivalent to host.docker.internal on mac, but that's not available on all platforms. Have added a comment to add context.